src/js/popup.js

```diff
@@ -503,6 +503,26 @@ function syncUISelections() {
 }
 
 /**
+* Get's the value of specific key within a url.
+* Temporary function for android while it doesn't support `browser_action.default_popup`
+*
+* Source: https://stackoverflow.com/a/901144/2662749
+* Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1330159
+*
+* @param  {String} key Key to get from url
+* @param  {String} url
+* @return {String}     Value of the key
+*/
+function getParameterByName(key, url) {
+  if (!url) { url = window.location.href; }
```

Comment:
I used that API before but didn't bother to read. Changed it know, should have read it before though 💤